src/communication/network_interface_sr.c

```diff
@@ -10972,12 +10978,23 @@ sflashback_get_loginfo (THREAD_ENTRY * thread_p, unsigned int rid, char *request
   ptr = or_unpack_int (ptr, &context.num_loginfo);
   ptr = or_unpack_int (ptr, &context.forward);
 
+  FLASHBACK_SET_REQUEST_TIME ();
+
   error_code = flashback_make_loginfo (thread_p, &context);
   if (error_code != NO_ERROR)
     {
       goto error;
     }
 
+  if (flashback_is_loginfo_generation_finished (&context.start_lsa, &context.end_lsa))
+    {
+      FLASHBACK_UNSET_MIN_LOG_PAGEID_TO_KEEP ();
+    }
+  else
+    {
+      FLASHBACK_SET_MIN_LOG_PAGEID_TO_KEEP (&context.start_lsa);
```

Comment:
I don't remember exactly, so I have a question: Is `context.start_lsa` <= `context.end_lsa` always true regardless of forward/backward?

Reply:
Yes, `context.start_lsa` <= `context.end_lsa` is always true.